pkgs/development/tools/misc/coreboot-toolchain/default.nix

```diff
@@ -0,0 +1,100 @@
+{ lib, stdenvNoCC, fetchurl, fetchgit,
+  gnumake, patch, zlib, git, bison,
+  flex, gnat11, curl, perl, binutils,
+  coreutils
+}:
+
+let
+  version_coreboot	= "4.14";
+
+  # from util/crossgcc/buildgcc
+  version_gmp		= "6.2.0";
+  version_mpfr		= "4.1.0";
+  version_mpc		= "1.2.0";
+  version_gcc		= "8.3.0";
+  version_binutils	= "2.35.1";
+  version_acpica	= "20200925";
+  version_nasm		= "2.15.05";
+
+  tar_name_gmp = "gmp-${version_gmp}.tar.xz";
+  tar_gmp = fetchurl {
+    url = "https://ftpmirror.gnu.org/gmp/${tar_name_gmp}";
+    sha256 = "09hmg8k63mbfrx1x3yy6y1yzbbq85kw5avbibhcgrg9z3ganr3i5";
+  };
+
+  tar_name_mpfr = "mpfr-${version_mpfr}.tar.xz";
+  tar_mpfr = fetchurl {
+    url = "https://ftpmirror.gnu.org/mpfr/${tar_name_mpfr}";
+    sha256 = "0zwaanakrqjf84lfr5hfsdr7hncwv9wj0mchlr7cmxigfgqs760c";
+  };
+
+  tar_name_mpc = "mpc-${version_mpc}.tar.gz";
+  tar_mpc = fetchurl {
+    url = "https://ftpmirror.gnu.org/mpc/${tar_name_mpc}";
+    sha256 = "19pxx3gwhwl588v496g3aylhcw91z1dk1d5x3a8ik71sancjs3z9";
+  };
+
+  tar_name_gcc = "gcc-${version_gcc}.tar.xz";
+  tar_gcc = fetchurl {
+    url = "https://ftpmirror.gnu.org/gcc/gcc-${version_gcc}/${tar_name_gcc}";
+    sha256 = "0b3xv411xhlnjmin2979nxcbnidgvzqdf4nbhix99x60dkzavfk4";
+  };
+
+  tar_name_binutils = "binutils-${version_binutils}.tar.xz";
+  tar_binutils = fetchurl {
+    url = "https://ftpmirror.gnu.org/binutils/${tar_name_binutils}";
+    sha256 = "01w6xvfy7sjpw8j08k111bnkl27j760bdsi0wjvq44ghkgdr3v9w";
+  };
+
+  tar_name_acpica = "acpica-unix2-${version_acpica}.tar.gz";
+  tar_acpica = fetchurl {
+    url = "https://acpica.org/sites/acpica/files/${tar_name_acpica}";
+    sha256 = "18n6129fkgj85piid7v4zxxksv3h0amqp4p977vcl9xg3bq0zd2w";
+  };
+
+  tar_name_nasm = "nasm-${version_nasm}.tar.bz2";
+  tar_nasm = fetchurl {
+    url = "https://www.nasm.us/pub/nasm/releasebuilds/${version_nasm}/${tar_name_nasm}";
+    sha256 = "1l1gxs5ncdbgz91lsl4y7w5aapask3w02q9inayb2m5bwlwq6jrw";
+  };
+
+  git_coreboot = fetchgit {
+    url = "https://review.coreboot.org/coreboot";
+    rev = "refs/tags/${version_coreboot}";
+    sha256 = "12ki8ffdnh1xir8j5z81rxlr9qhi7lk1mjq1n4yhh1nq1nsxnh6f";
```

Comment:
This is giving a different hash for me

Reply:
Right, for me too. One of the submodules might have changed. I will dig into that.

Reply:
I added fetchSubmodules to fetchgit and set it to false, since we don't need submodules anyway.

Reply:
It seems `leaveDotGit` has some problems which are the reason for the different hash. I switched to fetchurl for now.